src/main/sd/common.c

```diff
@@ -0,0 +1,105 @@
+#include "sd.h"
+#include "spi.h"
+
+#include "debug.h"
+#include "pin.h"
+
+#include <stdlib.h>
+#include <string.h>
+
+#define CRC7_POLYNOMIAL 0x9
+
+uint8_t calc_crc7(uint8_t* buffer, uint8_t count) // buffer with included byte for crc
```

Comment:
Why is crc here? We have a special module for that under src/crc

Reply:
crc removed